kratos/utilities/read_and_set_accessors_utilities.cpp

```diff
@@ -0,0 +1,67 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/read_and_set_accessors_utilities.h"
+
+namespace Kratos {
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void ReadAndSetAccessorsUtilities::ReadAndSetAccessors(
+    const Parameters MaterialData,
+    Properties& rProperty
+    )
+{
+    if (MaterialData.Has("accessors")) {
+        Parameters accessors = MaterialData["accessors"];
+
+        // Loop over the accessors list
+        for (auto iter = accessors.begin(); iter != accessors.end(); ++iter) {
+            auto accessor_param = accessors.GetValue(iter.name());
+            
+            // Table Accessor
+            if (accessor_param["accessor_type"].GetString() == "table_accessor") {
+                // Independent Variable
+                std::string input_var_name = accessor_param["properties"]["table_input_variable"].GetString();
+                Variable<double> *p_input_var = static_cast<Variable<double> *>(KratosComponents<VariableData>::pGet(input_var_name));
```

Comment:
```suggestion
               const Variable<double> *p_input_var = Variable<double> *KratosComponents<VariableData>::pGet(input_var_name);
```

you should not cast away the `const`